index.d.ts

```diff
@@ -124,6 +124,38 @@ declare module 'coinbase-pro' {
     active: boolean;
   };
 
+  export type PaymentMethod = {
+    id: string,
+    type: string,
+    name: string,
+    currency: CurrencyType,
+    primary_buy: boolean,
+    primary_sell: boolean,
+    allow_buy: boolean,
+    allow_sell: boolean,
+    allow_deposit: boolean,
+    allow_withdraw: boolean
+    limits: PaymentMethodLimits
+  };
+
+  export type PaymentMethodLimit = {
```

Comment:
Can you sort the definitions by use, so PaymentMethodLimitAmount first, then PaymentMethodLimit, PaymentMethodLimits and then PaymentMethod. That seems to be the style used in the rest of the file.

Reply:
👍